site/content/docs/5.0/forms/validation.md

```diff
@@ -351,14 +351,30 @@ If your form layout allows it, you can swap the `.{valid|invalid}-feedback` clas
 
 ## Customizing
 
-Validation states can be customized via Sass with the `$form-validation-states` map. Located in our `_variables.scss` file, this Sass map is looped over to generate the default `valid`/`invalid` validation states. Included is a nested map for customizing each state's color and icon. While no other states are supported by browsers, those using custom styles can easily add more complex form feedback.
+Validation states can be customized via Sass with the `$form-validation-states` map. Located in our `_variables.scss` file, this Sass map is looped over to generate the default `valid`/`invalid` validation states. Included is a nested map for customizing each state's color, icon, tooltip color and focus shadow color. While no other states are supported by browsers, those using custom styles can easily add more complex form feedback.
 
 Please note that we do not recommend customizing these values without also modifying the `form-validation-state` mixin.
 
 This is the Sass map from `_variables.scss`. Override this and recompile your Sass to generate different states:
 
 {{< scss-docs name="form-validation-states" file="scss/_variables.scss" >}}
 
+Maps of `$form-validation-states` can contain 3 more optional items for tooltip and focus colors. If not used it will take default value from `form-validation-state` mixin.
+
+Example of map configuration with all map items:
+
+```scss
+$form-validation-states: (
+  "valid": (
+    "color": $form-feedback-valid-color,
+    "icon": $form-feedback-icon-valid,
+    "tooltip-color": color-contrast($color),
+    "tooltip-bg-color": rgba($color, $form-feedback-tooltip-opacity),
+    "focus-box-shadow-color": rgba($color, $input-btn-focus-color-opacity)
+  )
+);
+```
+
```

Comment:
Let's nix this entirely. We don't do this really for other mixins if memory serves, and if we did, we should be using our `scss-docs` tag to do it.

```suggestion
```